freqtrade/freqtradebot.py

```diff
@@ -33,6 +34,17 @@
 logger = logging.getLogger(__name__)
 
 
+class CancelReason(Enum):
```

Comment:
There may be a better place to put this? We use these strings in tests as well, so this will prevent any typos 👍

Reply:
`constants.py`? 

I'm also not sure this should be a enum - since we're only interrested in the string portion (`.value`) of it, a dict may be better suited.